Leaderboard/logging.js

```diff
@@ -0,0 +1,36 @@
+import config from 'config';
+import winston from 'winston';
+import expressWinston from 'express-winston';
+
+const winstonConfig = config.get('Logger');
+const loggerConfig = {
```

Comment:
Most of these should probably be in the config file, but I was running into errors when I was doing that. Issue for later.